src/get-html.js

```diff
@@ -0,0 +1,111 @@
+/**
+ * Copyright 2015 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {startsWith} from './string';
+
+/** @type {!Array<string>} */
+const excludedTags = ['script', 'style'];
+
+/** @type {!Array<string>} */
+const allowedAmpTags = ['amp-accordion', 'amp-app-banner', 'amp-carousel',
```

Comment:
@lannka 
Sorry for the long wait for an answer.
`amp-accordion`, `amp-size text`, `amp-panel` - can contain page context (the menu items or just some text).
`amp-form`, `amp-app-banner`, `amp-selector` - can contain some calls to action. For example, buy this unique book for just $9.99!

From the attributes seem to be most useful 'alt', 'class', 'id', 'name', 'title'. They may contain information about content.
Other attributes may need us in the future.